docs/API/v3/registration-base-url-resource.md

```diff
@@ -0,0 +1,301 @@
+
+#### Registration page object
+
+The registration page object found in the registration index has the following properties:
+
+Name   | Type             | Required | Notes
+------ | ---------------- | -------- | -----
+@id    | string           | yes      | The URL to the registration page
+count  | integer          | yes      | The number of registration leaves in the page
+items  | array of objects | no       | The array of registration leaves and their associate metadata
+lower  | string           | yes      | The lowest SemVer 2.0.0 version in the page (inclusive)
+parent | string           | yes      | The URL to the registration index
+upper  | string           | yes      | The highest SemVer 2.0.0 version in the page (inclusive)
+
+The `lower` and `upper` bounds of the page object are useful when the metadata for a specific page version is needed.
+These bounds can be used to fetch the only the registration page needed. The version strings adhere to
+[NuGet's version rules](../../reference/package-versioning.md). The version strings are normalized and do not include
+build metadata.
+
+If the `items` property is not present in the registration page object, the URL specified in the `@id` must be used to
+fetch metadata about individual package versions. The `items` array is sometimes excluded from the page
+object as an optimization. If the number of versions of a single package IDs is very large, then the registration index
+document will be massive and wasteful to process for a client that only cares about a specific version or small range
+of versions.
+
+Note that is the `items` property is present, the `@id` property need not be used, since all of the page data is
+already inlined in the `items` property.
+
+Each item in the page object's `items` array is a JSON object representing a registration leaf and it's associated
+metadata.
+
+#### Registration leaf object in a page
+
+The registration leaf object found in a registration page has the following properties:
+
+Name           | Type   | Required | Notes
+-------------- | ------ | -------- | -----
+@id            | string | yes      | The URL to the registration leaf
+catalogEntry   | object | yes      | The catalog entry containing the package metadata
+packageContent | string | yes      | The URL to the package content (.nupkg)
+
+Each registration leaf object represents data associated with a single package version.
+
+#### Catalog entry
+
+The `catalogEntry` property in the registration leaf object has the following properties:
+
+Name                     | Type                       | Required | Notes
+------------------------ | -------------------------- | -------- | -----
+@id                      | string                     | yes      | The URL to document used to produce this object
+authors                  | string or array of strings | no       | 
+dependencyGroups         | array of objects           | yes      | The URL to the package content (.nupkg)
```

Comment:
Is `dependencyGroups` required? what if a package has no dependencies? Also, the Note for this entry does not seem to match?

Reply:
Nice catch. Note required.

Reply:
Fixed.